payment/datastore.go

```diff
@@ -62,6 +71,75 @@ func NewPostgres(databaseURL string, performMigration bool, dbStatsPrefix ...str
 	return nil, err
 }
 
+// CreateKey FIXME
+func (pg *Postgres) CreateKey(merchant string, encryptedSecretKey string, nonce string) (*Key, error) {
+	// interface and create an api key
+	var key Key
+	err := pg.DB.Get(&key, `
+			INSERT INTO api_keys (merchant_id, encrypted_secret_key, nonce)
+			VALUES ($1, $2, $3)
+			RETURNING *
+		`,
+		merchant, encryptedSecretKey, nonce)
+
+	if err != nil {
+		return nil, err
```

Comment:
`return nil, fmt.Errorf("failed to create key for merchant: %w", err)` would help debugging... This goes for all the datastore methods added.

Reply:
Added better debugging values here https://github.com/brave-intl/bat-go/pull/285/commits/9554365daf004cf5eef905c3f2d7cb0733ae2677